index.html

```diff
@@ -0,0 +1,66 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Alick's Portfolio</title>
+  <meta name="Alick Nyirenda" content="Alick Portfolio">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+  <header>
+    <p class="header-text">Alick Nyirenda </p>
+    <a href="#"><i class="header-icon fa fa-bars fa-lg"></i></a>
+  </header>
+  <main>
+  <section class="headline">
+    <h1 class="headline-text-1">Alick Nyirenda</h1>
+    <p class="headline-text-2">Software Developer</p>
+  </section>
+  <section class="headline-2">
+    <h3 class="headline-text-3">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me.</h3>
+    <div class="dividing-line"></div>
+    <div class="skills-container">
+      <div class="skill-item">
+        <p class="skill-item-title">Language</p>
+        <p class="skill-item-language">Javascript</p>
+        <p class="skill-item-language">Ruby</p>
+        <p class="skill-item-language">HTML</p>
+        <p class="skill-item-language">CSS</p>
+      </div>
+
+      <div class="skill-item">
+        <p class="skill-item-title">Frameworks</p>
+        <p class="skill-item-language">Bootstrap</p>
+        <p class="skill-item-language">Ruby on Rails</p>
+        <p class="skill-item-language">Express</p>
+        <p class="skill-item-language">Selenium</p>
+      </div>
+
+      <div class="skill-item">
+        <p class="skill-item-title">Skils</p>
+        <p class="skill-item-language">Terminal</p>
+        <p class="skill-item-language">Github</p>
+        <p class="skill-item-language">Gitlab</p>
+        <p class="skill-item-language">Codepen</p>
+      </div>
+    </div>
```

Comment:
- [ ] As this is a list, the technology categories ("Languages", "Frameworks" and "Skills") should be ordered using `ul`, each of them in a `li` tag. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list, use `ul` and each of them should be in a `li`. 

`ul` -  `li` orders the element semantically making it obvious to screen readers or assistive technologies to know they belong to the same group, this helps with proper accessibility and webpage optimisation.
Please try to use `ul` and `li` for semantic correctness whenever you are trying to list items. HTML has the ability to express lists of things, and it helps the Google robot, screen readers, and all manner of users who don't care solely about the presentation of the site understand your content better.

Reply:
All changes above made in the 3 commits below